mautic/mautic
# Updates emoji helper with the latest emoji

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | N
| New feature?  | Y 
| BC breaks?    | N
| Deprecations? | N
| Fixed issues  |  N

## Description

Background: Because Mautic uses utf8 for the database schema, 4 bit emojis cannot be saved. This helper converts the emojis back and forth. 

This PR simply updates the massive EmojiHelper with the latest emojis from php-emoji.

## Steps to test this PR

Create emails, landing pages, etc with emoji and make sure they save, are viewable, and editable
